contracts/test/TestModuleRelayer.sol

```diff
@@ -4,13 +4,13 @@ import "../modules/common/BaseModule.sol";
 import "../modules/common/RelayerModule.sol";
 
 /**
- * @title TestModule
- * @dev Basic test module.
- * @author Julien Niset - <julien@argent.xyz>
+ * @title TestModuleRelayer
```

Comment:
`TestModuleRelayerV2` was named that way to distinguish it from `TestModule` which was based on `RelayerModule` (V1). Now that we have only one version of the `RelayerModule`, we could just have the present module be named `TestModule` instead of `TestModuleRelayer`.

Reply:
Makes sense.